### DISCUSSION TITLE
Error: Format Bgra8UnormSrgb can't be multisampled

### DISCUSSION THREAD

**Discussion by mikeny07** [gfx-rs/wgpu#3825] [General]
When running a multi-sample app, it works for sample count = 1 or 4 as specified in WebGPU. However, wgpu supports sample count = 1, 2, 4, 8, 16 as mentioned in the docs. But I always got the following error when using sample count = 2, 8, or 16:

In Device::create_render_pipeline
      note: label = `Render Pipeline`
    Color state [0] is invalid
    Format Bgra8UnormSrgb can't be multisampled

How to fix this issue?

**Comment by Anti-Alias** [comment]
Same issue. Also, running this:

```
    fn print_format_features(adapter: &Adapter, format: TextureFormat) {
        let features = adapter.get_texture_format_features(format);
        let flags = features.flags;
        println!(
            "{format:?}: 1x: {}, 2x: {}, 4x: {}, 8x: {}",
            flags.sample_count_supported(1),
            flags.sample_count_supported(2),
            flags.sample_count_supported(4),
            flags.sample_count_supported(8)
        );
    }
```

Prints:

```
Bgra8Unorm: 1x: true, 2x: true, 4x: true, 8x: true
```

For the adapter I'm using.
I'm using this adapter in device/queue creation.

Win 10, explicitly selecting Vulkan API.

**Comment by breynard0** [comment]
I was running into this issue. The issue for me ended up being the render pass color attachment and command encoder were not in the same scope.
